src/components/NewCommentForm.tsx

```diff
@@ -1,38 +1,36 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
+import { addComment } from '../features/comments/commentsSlice';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 
 type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.